.github/workflows/dispatch.yml

```diff
@@ -7,11 +7,9 @@ on:
     branches: [ master, testing ]
 
 env:
-  # Number of repositories in a batch.
-  # Deployment jobs in the same batch are run sequentially.
-  # With ~180 repos, this means we'll run around 9 instances of the copy workflow in parallel.
-  # This is (hopefully) sufficient to prevent us from triggering GitHub Action's abuse detection mechanism.
-  MAX_REPOS_PER_WORKFLOW: 20
+  # The downstream workflow creates a matrix job per repository.
+  # 256 is the limit for number of matrix jobs a workflow can create.
+  MAX_REPOS_PER_WORKFLOW: 256
```

Comment:
That would mean that we're just running one job in parallel. I'd hope that we can avoid reducing the number that far.

Reply:
Yes, that was my intention. I'd want to see if we can get a fully successful run with a single job running at a time. Then, with the next release we could try to optimise it.

Reply:
Since it's not going out today, let me put this PR back into draft for now.